xbmc/interfaces/json-rpc/PlayerOperations.cpp

```diff
@@ -1555,6 +1555,71 @@ JSONRPC_STATUS CPlayerOperations::GetPropertyValue(PlayerType player, const std:
         break;
     }
   }
+  else if (property == "currentvideostream")
+  {
+    switch (player)
+    {
+    case Video:
+      if (g_application.m_pPlayer->HasPlayer())
+      {
+        result = CVariant(CVariant::VariantTypeObject);
+        int index = g_application.m_pPlayer->GetVideoStream();
+        if (index >= 0)
+        {
+          SPlayerVideoStreamInfo info;
+          g_application.m_pPlayer->GetVideoStreamInfo(index, info);
+
+          result["index"] = index;
```

Comment:
Maybe we could move the translation from `SPlayerVideoStreamInfo` to `CVariant` into a helper class as it's duplicate code.